### ISSUE TITLE
[Issue]: reboot via ui spams errors

### ISSUE THREAD

**Issue by phyzical** [jellyfin/jellyfin#11555]
### Please describe your bug

Rebooting via the ui on 10.9 is throwing the following

### Reproduction Steps

click reboot via ui

### Jellyfin Version

10.9.0

### Environment

```markdown
- OS: docker
- Linux Kernel:
- Virtualization:
- Clients:
- Browser: firefox
- FFmpeg Version:
- Playback Method:
- Hardware Acceleration:
- GPU Model:
- Plugins:
- Reverse Proxy:
- Base URL:
- Networking:
- Storage:
```

### Jellyfin logs

```shell
[17:24:04] [FTL] [12] Main: Error while starting server
System.InvalidOperationException: .NET runtime metrics are already being collected. Dispose() of your previous collector before calling this method again.
   at Prometheus.DotNetRuntime.DotNetRuntimeStatsCollector..ctor(ServiceProvider serviceProvider, CollectorRegistry metricRegistry, Options options)
   at Emby.Server.Implementations.ApplicationHost.Init(IServiceCollection serviceCollection)
   at Microsoft.Extensions.Hosting.HostBuilder.InitializeServiceProvider()
   at Microsoft.Extensions.Hosting.HostBuilder.Build()
   at Jellyfin.Server.Program.StartServer(IServerApplicationPaths appPaths, StartupOptions options, IConfiguration startupConfig)
[17:24:04] [INF] [12] Emby.Server.Implementations.ApplicationHost: Disposing CoreAppHost
[17:24:04] [INF] [12] Emby.Server.Implementations.ApplicationHost: Disposing PluginManager
[17:24:04] [INF] [12] Emby.Server.Implementations.AppBase.BaseConfigurationManager: Setting cache path: /config/cache
[17:24:04] [INF] [12] Emby.Server.Implementations.ApplicationHost: Loading assemblies
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by gnattu** [comment]
Will fix in 10.9.1

see  #11539

**Comment by phyzical** [comment]
too easy 👍 
duplicate
